dotCMS/src/main/java/com/dotmarketing/beans/transform/IdentifierTransformer.java

```diff
@@ -0,0 +1,48 @@
+package com.dotmarketing.beans.transform;
+
+import com.dotcms.util.transform.DBTransformer;
+import com.dotmarketing.beans.Identifier;
+import com.dotmarketing.business.DotStateException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * DBTransformer that converts DB objects into Identifier instances
+ */
+public class IdentifierTransformer implements DBTransformer {
+    final List<Identifier> list;
+
+
+    public IdentifierTransformer(List<Map<String, Object>> initList){
+        List<Identifier> newList = new ArrayList<>();
+        if (initList != null){
+            for(Map<String, Object> map : initList){
+                newList.add(transform(map));
+            }
+        }
+
+        this.list = newList;
+    }
+
+    @Override
+    public List<Identifier> asList() throws DotStateException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.dotmarketing.business.DotStateException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)